tensorflow/benchmark_convnet_model_tf.py

```diff
@@ -0,0 +1,460 @@
+"""
+based on https://github.com/tensorflow/models/blob/master/official/resnet/resnet_model.py
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+
+import numpy as np
+import paddle.v2 as paddle
+import paddle.v2.fluid.profiler as profiler
```

Comment:
In the TensorFlow, need to remove `paddle.v2.fluid.profiler`.

Reply:
done